front/style/pages/_referent.scss

```diff
@@ -1,4 +1,184 @@
 .referent {
+
+    .manager-header {
+        background-color: #f6f9fc;
```

Comment:
Can this color be a variable?

Reply:
yes i still need to add the new variables colors, its wip